tools/export-analyzer/main.go

```diff
@@ -45,9 +49,44 @@ func main() {
 		log.Fatalf("error unmarshaling export file: %v", err)
 	}
 
+	// Do some basic data validation.
+	if !*quiet {
+		checkExportFile(keyExport)
+	}
+
 	prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
 	if err != nil {
 		log.Fatalf("error pretty printing export: %v", err)
 	}
 	log.Printf("%v", string(prettyJSON))
 }
+
+func checkExportFile(export *exportpb.TemporaryExposureKeyExport) {
+	now := time.Now().UTC()
+	floor := model.IntervalNumber(now.Add(-14 * 24 * time.Hour))
```

Comment:
nit: could define 14 and 16 (below) as self-describing constants up above?

Reply:
Will do in follow up